cozy_management/couchdb.py

```diff
@@ -239,7 +239,7 @@ def get_database_dir():
     import ConfigParser
     result = {}
     file_list = helpers.cmd_exec('find /usr/local/etc/couchdb /etc/couchdb -type f -name "*.ini"')
-    file_list = list(s.rstrip('\n') for s in file_list['stdout'])
+    file_list = list(s for s in file_list['stdout'].rstrip('\n').split('\n'))
```

Comment:
I actually used the same variable already used in the list comprehension
(if that is the right name).

Le mar. 26 janv. 2016 20:01, Frank Rousseau <notifications@github.com> a
écrit :

> In cozy_management/couchdb.py
> <https://github.com/cozy/python_cozy_management/pull/4#discussion_r50882067>
> :
>
> > @@ -239,7 +239,7 @@ def get_database_dir():
> >      import ConfigParser
> >      result = {}
> >      file_list = helpers.cmd_exec('find /usr/local/etc/couchdb /etc/couchdb -type f -name "*.ini"')
> > -    file_list = list(s.rstrip('\n') for s in file_list['stdout'])
> > +    file_list = list(s for s in file_list['stdout'].rstrip('\n').split('\n'))
>
> Please avoid single letter variable. It's horrible to maintain (poor
> meaning and hard to perform renaming)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cozy/python_cozy_management/pull/4/files#r50882067>.
>